src/components/ContentPreview/Header.scss

```diff
@@ -5,8 +5,9 @@
 
     .be & .bcpr-bcow-btn,
     .be & .btn-plain.bcpr-btn {
-        line-height: 0;
-        padding: 0 15px 0 5px;
+        display: flex;
```

Comment:
Double check how this looks with open with!

Reply:
<img width="818" alt="screen shot 2018-10-04 at 3 02 12 pm" src="https://user-images.githubusercontent.com/17791289/46505727-d97c2f00-c7e6-11e8-9fd9-b01f5c95f927.png">